fluent-bundle/src/bundle.rs

```diff
@@ -127,7 +130,7 @@ pub struct FluentBundle<R> {
     pub locales: Vec<LanguageIdentifier>,
     pub(crate) resources: Vec<R>,
     pub(crate) entries: HashMap<String, Entry>,
-    pub(crate) plural_rules: PluralRules,
+    pub(crate) intls: Rc<RefCell<IntlLangMemoizer>>,
```

Comment:
you should make sure that going through a hashmap lookup (pretty cheap) each time you need the plural rules is not impacting perf too much